Src/BootCamp.Chapter/ConsoleArgumentsParser.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public static class ConsoleArgumentsParser
+    {
+        public static void ArgumentsParser(string[] args)
```

Comment:
Function name should be a **verb** (not ok) and class name should be a **noun** (ok)